zbar_lite/README.md

```diff
@@ -0,0 +1,81 @@
+# This package is a part of ZBAR, only supports image related functions. For now , it is only designed for Python.
+
+#How to build and install
+**It is recommended for you to run cmd below first.**
+```
+cd zbar
+autoreconf -vfi
+./configure --without-java --without-gtk --without-qt  --without-imagemagick  --disable-video --without-python
+```
+the ``host`` you should fill with something according to your own compiler.
+
+then run
+```
+sh preparation.sh
+```
+
+before build, you should make sure that your gcc compiler is fit with your OS. 
+
+you can install gcc build env from `https://sourceforge.net/projects/mingw-w64/files/` for windows
+
+Notice that mingw64 and mingw-w64 is not the same thing.
+
+I select `x86_64-posix-seh-rev0` to build my wheel on Windows.
+
```

Comment:
It just provides an example for Windows. Also, other OSes will be supported in the future.
I build in this way successfully on WINDOWS and LINUX.

Reply:
> > Is this meant to run just on Windows? What about the other supported OSes?

> It just provides an example for Windows. 

The text at `README.md` should be generic enough to cover all supported OSes. If you want to add some comments specific for Windows, please create a separate session for that, but then it would make sense to have an OS-specific section for Linux and MacOS - in order for the document to be symetric :-) 

> Also, other OSes will be supported in the future. I build in this way successfully on WINDOWS and LINUX.